{"name": "test_valid_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'Session' object has no attribute 'my_test_result'", "trace": "self = <allure_commons._allure.fixture object at 0x0000020B28D9A210>, args = (), kwargs = {}\n\n    def __call__(self, *args, **kwargs):\n        self.parameters = func_parameters(self._fixture_function, *args, **kwargs)\n    \n        with self:\n>           return self._fixture_function(*args, **kwargs)\n\n..\\afwb70\\venv\\Lib\\site-packages\\allure_commons\\_allure.py:221: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <test_scripts.test_valid_login.TestValidLogin object at 0x0000020B29949290>, request = <SubRequest 'allure_report' for <Function test_valid_login>>\n\n    @pytest.fixture(autouse=True)\n    def allure_report(self, request):\n        yield\n>       print(request.session.my_test_result)\nE       AttributeError: 'Session' object has no attribute 'my_test_result'\n\ngeneric\\base_setup.py:80: AttributeError"}, "attachments": [{"name": "stdout", "source": "86234caf-3a27-409c-9519-08921471c230-attachment.txt", "type": "text/plain"}], "start": 1679917249520, "stop": 1679917253293, "uuid": "1f1ec0c1-6cd4-43de-a976-fb216acc2192", "historyId": "ab84b7315ad1f8a783ebfebfd69e768d", "testCaseId": "1311271321bd346241d5e7f75a3a7f0a", "fullName": "test_scripts.test_valid_login.TestValidLogin#test_valid_login", "labels": [{"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_scripts"}, {"name": "suite", "value": "test_valid_login"}, {"name": "subSuite", "value": "TestValidLogin"}, {"name": "host", "value": "BHANU-LAPTOP"}, {"name": "thread", "value": "55600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_scripts.test_valid_login"}]}